packages/api-client/src/fragments/attributeValue.fragment.ts

```diff
@@ -0,0 +1,18 @@
+import gql from 'graphql-tag';
+
+export const AttributeValueFragment = gql`
+  fragment AttributeValue on AttributeValue {
+    items: {
```

Comment:
Please test all the fragments and queries in the playground first. 
This fragment is wrong and won't work as is.

`AttributeValue` is not valid with the `items` field, it should be `AttributeValuePaginatedResponse` or whatever the type in the schema is.

Also please include all the rest of the paginated fields like `page`, `offset` , `itemsFound` etc. etc.